mango_base/mango/dashboards/time_series_utils/data_loader.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+import streamlit as st
```

Comment:
Review if this script .py must be here or in the time-series-dashboard branch.
@antoniogonzalezsuarez